src/__tests__/helpers/mockFetch.js

```diff
@@ -0,0 +1,9 @@
+import fetchMock from 'node-fetch';
+
+export const mockFetch = html => {
```

Comment:
Please don't extract `mockFetch` in this PR.
We'll do this in another PR, to keep this PR as clean as possible. 🙂